requirements/base/base.txt

```diff
@@ -7,3 +7,4 @@ textdistance >= 4.0, < 5.0
 tqdm >= 4.0, < 5.0
 typing-extensions >= 4.0, < 5.0
 requests
+tenacity >= 9.0
```

Comment:
@RafalSkolasinski also suggested using tenacity.

Good to keep it in mind when reviewing existing retry logic or adding new one 👍